src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -44,10 +44,11 @@ public static class DataTestUtility
         public static bool EnclaveEnabled { get; private set; } = false;
         public static readonly bool TracingEnabled = false;
         public static readonly bool SupportsIntegratedSecurity = false;
-        public static readonly bool SupportsFileStream = false;
         public static readonly bool UseManagedSNIOnWindows = false;
         public static readonly bool IsAzureSynapse = false;
         public static Uri AKVBaseUri = null;
+        public static string FileStreamDirectory = null;
+        private static string _fileStreamDBName = null;
```

Comment:
I believe we could manage this value inside the related tests and taking this field out of here.

Reply:
I added it here so we create only 1 such DB at a time.

Reply:
I've modified the design to take this to test level.